tests/integration/targets/ec2_instance/roles/ec2_instance/tasks/termination_protection.yml

```diff
@@ -105,13 +161,31 @@
     - name: Check return value
       assert:
         that:
-          - "{{ set_termination_protection_results.changed }}"
-          - "{{ not set_termination_protection_results.changes[0].DisableApiTermination.Value }}"
+          - set_termination_protection_results.changed
+
+    - name: Get info on termination protection
+      command: aws ec2 describe-instance-attribute --attribute disableApiTermination --instance-id {{ instance_id }}
+      environment:
+        AWS_ACCESS_KEY_ID: "{{ aws_access_key }}"
+        AWS_SECRET_ACCESS_KEY: "{{ aws_secret_key }}"
+        AWS_SESSION_TOKEN: "{{ security_token | default('') }}"
+        AWS_DEFAULT_REGION: "{{ aws_region }}"
+      register: instance_termination_check
```

Comment:
It feels wrong that the only way we can tell the state of the attributes is a CLI call...

Reply:
I don't disagree, but right now we don't have anything that gets info on this.  There's no generic "get all attributes", you have to describe them one at a time so not so simple to sneak into the module.   https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_instance_attribute

That would make a good info module or lookup, but IMO that's out of scope for this PR.